src/client/resources/conversions.ts

```diff
@@ -67,3 +67,63 @@ export function createMigCluster(
     },
   };
 }
+
+export function createMigStorage(
```

Comment:
+1, I think it's important to have functions that accept the params and spit back the actual kube object.